Code-4-Community/lucys-love-bus-backend
# added public announcements route

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/XXXXXXXX/pulses/XXXXXXX)

This adds a new route for public announcements needed for announcements page.

## This PR

Exposes GET request for all announcements to public route.

## Screenshots

[Endpoint picture](https://ibb.co/6bMH701)

## Verification Steps

Start the server and verify that you are able to access the new announcements route. I would suggest adding test data with pgAdmin. Also verify that the protected announcement routes are still working as expected.
